src/test/java/bio/terra/app/controller/SnapshotAccessRequestApiControllerTest.java

```diff
@@ -266,6 +289,17 @@ void testDeleteSnapshotRequestGroupMember() throws Exception {
     verifyAuthorization(IamResourceType.SNAPSHOT_BUILDER_REQUEST, id, IamAction.APPROVE);
   }
 
+  @Test
+  void testDeleteSnapshotRequestGroupMemberNotCreated() throws Exception {
+    UUID id = UUID.randomUUID();
+    String memberEmail = "user@gmail.com";
+    SnapshotAccessRequestResponse accessRequest =
+        new SnapshotAccessRequestResponse().createdSnapshotId(null);
+    when(snapshotBuilderService.getRequest(id)).thenReturn(accessRequest);
+    mvc.perform(delete(DELETE_MEMBERS_ENDPOINT, id, memberEmail));
+    verify(snapshotBuilderService, never()).deleteGroupMember(id, memberEmail);
+  }
```

Comment:
For these tests, I wasn't sure the best way to mock and/or assert that the Illegal Argument Exception actually gets thrown so I opted for just checking that the rest of the method never gets called. I'm not sure if that's the best approach, looking for feedback or suggestions there

Reply:
You could do something like this:

Reply:
You could try something like this to check for the exact exception type 
    mvc.perform(delete(DELETE_MEMBERS_ENDPOINT, id, memberEmail))
        .andExpect(
            result ->
                assertTrue(
                    result.getResolvedException() instanceof IllegalArgumentException));

Reply:
Or if you want to test more broadly that its a 400 error 
 mvc.perform(delete(DELETE_MEMBERS_ENDPOINT, id, memberEmail))
        .andExpect(status().is4xxClientError());